src/Model/GroundWaterFlow/gwf3sfr8.f90

```diff
@@ -2804,6 +2806,16 @@ subroutine sfr_set_stressperiod(this, n, ichkustrm)
         call read_value_or_time_series_adv(text, ii, jj, bndElem, this%packName,     &
                                            'BND', this%tsManager, this%iprpak,   &
                                            'DIVFLOW')
+        !
+        ! -- if diversion cprior is 'fraction', ensure that 0.0 <= fraction <= 1.0
+        cp = this%divcprior(ii)
+        divq = this%divflow(ii) 
+        if (cp == 'FRACTION' .and. (divq < 0.0 .or. divq > 1.0)) then
```

Comment:
Should probably use the constants DZERO and DONE instead of 0.0 and 1.0.

Reply:
Thanks for the pointer @langevin-usgs, have patched and PR is now rerunning

Reply:
@langevin-usgs I looked at the failure and it occurred with one of the models in modflow6-testmodels.git, specifically one that resides in mf5to6:
![mf5to6](https://user-images.githubusercontent.com/3236576/108109703-1b6c3f80-7047-11eb-88a8-ea7afe82d0de.png)
Not being sure if the error was anomalous, I tried running the model locally after running the conversion program:
![mf5to6_error](https://user-images.githubusercontent.com/3236576/108109786-39d23b00-7047-11eb-90d5-b0af2063a5cf.png)
It doesn't seem like this error is related to the change made in this PR? Since the first commit on this PR passed, then changing 0.0 -> DZERO in the second commit caused it to fail, I'm wondering if we should try and get it to run again?